specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-07-01/examples/CheckNameAvailability_NotAvailable.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "77777777-b0c6-47a2-b37c-d8e65a629c18",
+    "resourceGroupName": "resGroupBar",
+    "api-version": "2021-06-01",
```

Comment:
do you need to update the api-version here?

Reply:
It seems this issue happens in all your sample files.

Reply:
updated the version in all swagger and example jsons